src/Microsoft.SqlTools.Dmp.Contracts/Hosting/Error.cs

```diff
@@ -0,0 +1,24 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+
+namespace Microsoft.SqlTools.Dmp.Contracts.Hosting
+{
+    /// <summary>
+    /// Defines the message contract for errors returned via SendError.
```

Comment:
Minor: you can use `<see cref="Microsoft.SqlTools.Dmp..RequestContext.SendError" />` syntax to reference the actual API method call. When the XML Doc is built it'll have the right hyperlink to navigate to referenced code.

Reply:
I'm not sure how that works for code in different projects. ReSharper isn't happy about it, and I'm not sure what the XML Doc generator will turn that into. But I'll implement it anyways.